packages/rpc-parsed-types/src/sysvar-accounts.ts

```diff
@@ -0,0 +1,79 @@
+import type { Blockhash, Epoch, Slot, StringifiedBigInt, UnixTimestamp } from '@solana/rpc-types';
+
+import { RpcParsedType } from './rpc-parsed-type';
+
+type FeeCalculator = Readonly<{
+    lamportsPerSignature: StringifiedBigInt;
+}>;
+
+type JsonParsedClockAccount = Readonly<{
+    slot: Slot;
+    epoch: Epoch;
+    epochStartTimestamp: UnixTimestamp;
+    leaderScheduleEpoch: Epoch;
+    unixTimestamp: UnixTimestamp;
+}>;
```

Comment:
These made me think of possibly rolling a helper somewhere, maybe in the main library, for `getClock()`. Thoughts?

Reply:
#2041